### DISCUSSION TITLE
Does DB2 with ODBC require paid DBeaver?

### DISCUSSION THREAD

**Discussion by guslipkin** [dbeaver/dbeaver#34149] [Q&A] [answered]
I am confused by the documentation. The community edition documentation indicates that DB2 is a supported database. However, this page seems to indicate that DB2 is only supported on a paid tier of DBeaver. I would like to use an ODBC driver I have installed separately, and not a DBeaver provided driver or a JDBC driver. Is this something I can do on the community edition, or do I need to upgrade?

Community Edition source: https://dbeaver.io/about/
This page: https://dbeaver.com/docs/dbeaver/ODBC-JDBC-Driver/

In addition, the DBeaver Documentation for the DB2 driver makes no mention of which tiers are supported: https://dbeaver.com/docs/dbeaver/Database-driver-IBM-Db2/

**Comment by serge-rider** [comment, marked as answer]
DB2 JDBC driver is included in DBeaver Community. It works in most cases.

ODBC drivers support presents only in commercial versions of DBeaver (any ODBC drivers, not just DB2 ones).

So if you need ODBC driver (for any reason) then you need to upgrade to commercial version.
